FAQ.md

```diff
@@ -94,22 +94,22 @@ You need 3 packages (applications) enabled (use search in NetGuard to find them
 Since the Google Play™ store app has a tendency to check for updates or even download them all by itself (even if no account is associated),
 one can keep it in check by enabling "*Allow when device in use*" for all 3 of these packages.
 Click on the down arrow on the left side of an application name and check that option,
-but leave the network icons set to red (hence blocked).The little human icon will appear for those packages.
+but leave the network icons set to red (hence blocked).  The little human icon will appear for those packages.
 
-Note that NetGuard does not require any Google service to be installed.
+Note that NetGuard does *not* require any Google service to be installed.
 
 <a name="FAQ9"></a>
 **(9) Why is the VPN service being restarted?**
 
 The VPN service will be restarted when you turn the screen on or off and when connectivity changes (Wi-Fi, mobile)
-to apply the rules with the conditions '*Allow when screen is on*' and '*Block when roaming*'.
+to apply the rules with the conditions *'Allow when screen is on'* and *'Block when roaming'*.
 
 See [here](http://forum.xda-developers.com/showpost.php?p=65723629&postcount=1788) for more details.
 
 <a name="FAQ10"></a>
 **(10) Will you provide a Tasker plug-in?**
 
-If disabling NetGuard is allowed to Tasker, any application can disabled NetGuard too.
+No, because if Tasker is allowed to disable NetGuard, any application can disabled NetGuard.
```

Comment:
You forgot to change "any application can disabled NetGuard" to "any application can disable NetGuard"